tools/driver-cxx/driver-cxx.cpp

```diff
@@ -292,6 +293,10 @@ int main(int argc, char **argv) {
 
   mull::IDEReporter ideReporter;
   ideReporter.reportResults(*result, rawConfig, metrics);
+
+  mull::MutationTestingElementsReporter mutationTestingElementsReporter(junkDetector.getMutationPointsRanges());
```

Comment:
Does it compile? I cannot find this method, and the reporter takes the ASTStorage now.